sources/packages/web/src/services/http/dto/SupportingUser.dto.ts.ts

```diff
@@ -0,0 +1,52 @@
+import { ContactInformation, SupportingUserType } from "@/types";
```

Comment:
Please remove the additional ".ts" in the file name.

Reply:
oh ya